packages/core/src/framebuffer/FramebufferSystem.js

```diff
@@ -216,14 +216,15 @@ export class FramebufferSystem extends System
      * @param {Number} g - Green value from 0 to 1
      * @param {Number} b - Blue value from 0 to 1
      * @param {Number} a - Alpha value from 0 to 1
+     * @param {Number} mask - Bitwise OR of masks that indicate the buffers to be cleared, by default COLOR
```

Comment:
Type should be `PIXI.BUFFER_BITS` and the default value should be provided for mask: `[mask=PIXI.BUFFER_BIT.COLOR]`